rasa/core/channels/instagram.py

```diff
@@ -0,0 +1,334 @@
+import hashlib
+import hmac
+import inspect
+import logging
+from typing import Text, Dict, Any, Callable, Awaitable, Optional, List
+
+from fbmessenger import MessengerClient
+from fbmessenger.elements import Text as InstaText
+from fbmessenger.attachments import Image
+from fbmessenger.quick_replies import QuickReplies, QuickReply
+from fbmessenger.sender_actions import SenderAction
+from rasa.core.channels.channel import UserMessage, OutputChannel, InputChannel
+from sanic import Blueprint, response
+from sanic.request import Request
+from sanic.response import HTTPResponse
+
+logger = logging.getLogger(__name__)
+
+
+class Messenger:

```

Comment:
This class requires a docstring (see `facebook.py` for an example)

Reply:
In addition, the class name is the same as the one in `facebook.py`

Reply:
I am on it, changing it to InstaMessenger and adding docstring wherever required